packages/react/src/page-alert/docs/overview.mdx

```diff
@@ -118,70 +118,129 @@ Use the `onClose` prop to make the alert dismissable. Ensure Page alerts that ar
 
 _NOTE: When focusing an alert after closing a `Drawer`, use the [focusElementOnClose prop in the Drawer component](/components/drawer#focus-custom-element-on-close)._
 
-In the event you need to focus the alert, you can do so in one of two ways:
+In the event you need to focus the alert, you can do so in one of three ways:
 
-1. [Manually set focus](#manually-set-focus).
-2. [Automatically focus on mount](#automatically-focus-on-mount).
+1. [Automatically focus on mount](#automatically-focus-on-mount).
+2. [Automatically focus on update](#automatically-focus-on-update).
+3. [Manually set focus](#manually-set-focus).
 
-### Manually set focus
+### Automatically focus on mount
 
-Press the "Focus the alert" button below to set focus on the alert. To achieve this, we set the `tabIndex` to `-1`. This will allow programmatic focusing and will render a focus ring to aid accessibility.
+Press the "Toggle alert" button below to mount/unmount the alert. Notice that each time the alert is mounted it will be focused. Here we use the `focusOnMount` boolean prop to automatically handle the focusing.
 
 ```jsx live
 () => {
-	const alertRef = React.useRef(null);
+	const [showAlert, setShowAlert] = React.useState(false);
 
 	return (
 		<Stack gap={2}>
 			<Box alignSelf="start">
```

Comment:
This doesn't actually do anything.
```suggestion
			<Box>
```

Reply:
Your Focus custom element on close Drawer example also has this unnecessary prop